tests/admin_widgets/tests.py

```diff
@@ -898,6 +899,54 @@ class DateTimePickerAltTimezoneSeleniumIETests(DateTimePickerAltTimezoneSelenium
 
 
 @override_settings(PASSWORD_HASHERS=['django.contrib.auth.hashers.SHA1PasswordHasher'],
```

Comment:
you can put PASSWORD_HASHERS on the next line here (the existing style is not preferred).